package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hof-util-reqres",
+  "version": "1.0.0",
+  "description": "Test helper to create dummy request/response objects for testing middleware methods",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/UKHomeOfficeForms/hof-util-reqres.git"
+  },
+  "author": "",
```

Comment:
I tried to find any consistency across our modules in this regard. There is none, so I went for the same as the first module I found which had an author defined.